### DISCUSSION TITLE
Hey Hey

### DISCUSSION THREAD

**Discussion by atebyagrue** [evennia/evennia#2647] [Greetings & Introductions]
I found Evennia after starting to develop my own Python-based MUD; saw that everything I had planned was already done & decided to give it a spin. 
I've been using it as an online character sheet and generator for my cyberpunk tabletop rpg players, while secretly building a game world in the background for them to be able to solo play their characters.
Came from the world of CircleMUD and DIKU, so a lot of this is real familiar to me. I've been playing text-based games since the 80's and been working on my own MUD's for a while now. I'm an  developer by trade and looking forward to contributing to the project!

**Comment by Tegiminis** [comment]
Welcome to the project!

**Comment by Griatch** [comment]
Welcome! A fun use case to use Evennia as a tie-in with a TTRPG :)
